app/models/pension/charges/UnauthorisedPaymentsViewModel.scala

```diff
@@ -37,7 +37,7 @@ case class UnauthorisedPaymentsViewModel(unauthorisedPaymentsQuestion: Option[Bo
                                          pensionSchemeTaxReference: Option[Seq[String]] = None) {
 
   def encrypted()(implicit secureGCMCipher: SecureGCMCipher, textAndKey: TextAndKey): EncryptedUnauthorisedPaymentsViewModel =
-    EncryptedUnauthorisedPaymentsViewModel(
+      EncryptedUnauthorisedPaymentsViewModel(
```

Comment:
The `unauthorisedPaymentsQuestion ` field can be removed now.

And I would strongly recommend introducing a `unauthorisedPaymentsQuestion' function in this class, which will return the current value as derived from `surchargeQuestion` and `noSurchargeQuestion`.